README.md

```diff
@@ -1,31 +1,31 @@
-  * In the second to last `it` function call in `test/indexTest.js` we test that `console.log` is called with our `driver` variable. This is a task you may find yourself doing a lot. Using `console.log` to see the value of a variable. If you pass the `variable` driver to `console.log` your test should be passing. More to the point, if you open up your browser's console (by pressing cmd+shift+c on a mac), you can see your corresponding data logged.
-  * The final `it` function call in this lab, the test asks you to practice passing multiple arguments to `console.log`. The first argument is the `driver` variable, and the second argument should be the string `is the driver variable name`.
+  * If you look at the first `it` function call, the tests ask that `invokes console.log() with the string "this code was called"`. Then, the next line says `spy` to be `calledWithExactly` the string `this code was called`. To pass this test, we can write `console.log('this code was called')` inside of our `index.js` file. Yes, I know we are a little confused. What is a spy? It's ok if you walk away from the course not fully knowing. Just understand that this `spy` thing helps us test that `console.log` is called, and what the arguments are. So whenever you see `expect(spy.calledWithExactly('this code was called')).to.be.true` what we really mean is that we expect `console.log` to be called with the argument `this code was called`. For more details on what the `spy` is doing, see our "note about spies" below.
+  * In the second to last `it` function call in `test/indexTest.js` we test that `console.log` is called with our `driver` variable. This is a task you may find yourself doing a lot. Using `console.log` to see the value of a variable. If you pass the `variable` driver to `console.log` your test should be passing. More to the point, if you open up your browser's console (by pressing cmd+shift+c on a Mac, ctrl+shift+j on chrome on a PC), you can see your corresponding data logged.
+  * In the final `it` function call in this lab, the test asks you to practice passing multiple arguments to `console.log`. The first argument is the `driver` variable, and the second argument should be the string `is the driver variable name`.
```

Comment:
Do we need to give them PC instructions? None of our other lessons cater to PC that I'm aware of since it's a whole other beast in some cases.

Reply:
I tend to like to add the PC instructions just in case. We don't require skills course students to have a mac as far as I know, and I think this lesson is used other places, so it's nice to have.